remote-api/src/test/java/org/alfresco/rest/api/impl/TagsImplTest.java

```diff
@@ -302,4 +314,33 @@ private static Tag createTagWithNodeRef(final String tagName)
             .tag(tagName)
             .create();
     }
+
+    @Test
+    public void testAddTagsToNode()
+    {
+        final List<String> tagNames = List.of("tag1","tag2");
+        final List<Tag> tagsToCreate = createTags(tagNames);
+        given(taggingServiceMock.addTags(any(), any())).willAnswer(invocation -> createTagAndNodeRefPairs(invocation.getArgument(1)));
+        final List<Tag> actualCreatedTags = objectUnderTest.addTags(nodesMock.getNode(any()).getNodeId(),tagsToCreate);
+        then(taggingServiceMock).should().addTags(TAG_NODE_REF, tagNames);
+        final List<Tag> expectedTags = createTagsWithNodeRefs(tagNames).stream().peek(tag -> tag.setCount(1)).collect(Collectors.toList());;
+        assertThat(actualCreatedTags)
+                .isNotNull()
+                .isEqualTo(expectedTags);
+    }
+
+    @Test
+    public void testAddTagsToNodeWithResponseNotIndexed()
```

Comment:
This test is now identical to the previous test. Please fix the tests to do what they're meant to do.